pulser/tests/test_waveforms.py

```diff
@@ -194,6 +194,13 @@ def test_blackman():
     wf2 = BlackmanWaveform(duration + 1, area)
     assert np.max(wf2.samples) < np.max(wf.samples) <= max_val
 
+    # Same but with a negativa max value
```

Comment:
Tiny typo on `negativa` :wink:

```suggestion
    # Same but with a negative max value
```

Reply:
Nice catch, cheers!